aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java

```diff
@@ -109,7 +109,7 @@ public HttpIndexReader(String geoserver) {
     }
 
 
-    public String getLatestTimeStep(String layer, String timeField) throws AggregationException {
+    public String getLatestTimeStep(String layer, String timeField) throws AggregationException, TimeNotSupportedException {
```

Comment:
The different possible outcomes of this method are a bit confusing. It's hard to work out from reading what it means if you get a null returned vs an AggregationException vs a TimeNotSupportedException. I don't actually have any ideas for fixing that though...

Reply:
I considered some alternatives - but this seemed like roughly the best way to do it.  I have added some comments in the code as to why it was implemented this way.